farms/static/css/snek.css

```diff
@@ -0,0 +1,123 @@
+:root {
+  --purple: #5b57a6;
+  --green: #00b25a;
+  --orange: #f3703a;
+  --yellow: #ffc519;
+  --red: #e11d43;
+}
+body {
+  font-family: 'PT Sans', sans-serif;
+}
+h1 {
+  color: var(--green);
+  font-size: 2rem;
+}
+.container {
+  max-width: 40em;
```

Comment:
Is there any reason why you're swapping between using 'em' and 'rem' units? I noticed that nothing was defined in `html {}` that gives 'rem' its initial scale (EG: `body: 62.5%` so that 1rem ~= 10px)? if so, how do you determine what the em unit is basing its width off of? 

(Sorry, i've only really stuck with ems before. let me know if this is more of a google question!)

Reply:
The difference between `em` and `rem` is important for the sneks but pretty arbitrary for the `container` here (in fact this might be a merge conflict artifact... this file is a bit of a mess!)

Each snektion is defined in `em`, so the way to make a ｌａｒｇｅ　ｂｏｉ or a ˢᵐᵒˡ ᵇᵒᶦ is to change the font size in a parent element. I've done that in `rem`, just working from the default HTML size (because I don't really care about how many pixels something is, just how big it is relative to other things).

Reply:
I am wondering how long the "large boi / smol boi" comment here took :joy: I appreciate your dedication

Reply:
I have added full-width romaji to my languages for ease of ｌａｒｇｅｎｅｓｓ

<img width="256" alt="Screen Shot 2019-08-09 at 12 59 59 PM" src="https://user-images.githubusercontent.com/1615322/62746820-a315dd00-baa5-11e9-9c4c-5cae38f4aa8f.png">